demo/add_tenant.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) 2019 by eHealth Africa : http://www.eHealthAfrica.org
+#
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+set -Eeuo pipefail
+
+source .env
+source ./scripts/aether_functions.sh
+
+echo_message "Adding tenant $1..."
+echo_message "Starting kong..."
+start_container kong $KONG_INTERNAL
+echo_message "Starting keycloak..."
+start_container keycloak "${KEYCLOAK_INTERNAL}/auth"
+echo_message "Starting other services..."
```

Comment:
You don't need to start the other containers while you are adding them to keycloak/kong ;-)

Reply:
True, but I need ES, Kafka and ES up. I supposed I could cut it down to strictly what's required, but I assume the next step after adding a tenant will be to try to use it, so why not have it all up?

Reply:
you can always call `deomo/up.sh` outside this script and not wait 20 seconds every time you execute it...

```bash
# prepare the playground
./demo/add_tenant.sh ehealth
./demo/add_tenant.sh cdc
./demo/add_tenant.sh eoc

# start playing
./demo/up.sh

# game is over
./demo/stop.sh
```

:football: :tennis: :soccer:

Reply:
That's true, I'll Just add a note about services should be running before you start.